README.md

```diff
@@ -37,79 +30,136 @@ Flyte is an **open-source orchestrator** that facilitates building production-gr
-:mag: **Cache outputs**: Cache task outputs by passing `cache=True` to the task decorator. <br />
-:triangular_flag_on_post: **Intra-task checkpointing**: [Checkpoint progress](https://docs.flyte.org/projects/cookbook/en/latest/auto_examples/advanced_composition/checkpoint.html) within a task execution. <br />
-:earth_americas: **Multi-tenancy**: Multiple users can share the same platform while maintaining their own distinct data and configurations. <br />
-:alarm_clock: **Timeout**: Define a timeout period, after which the task is marked as failure. <br />
-:lock: **Immutability**: Immutable executions help ensure reproducibility by preventing any changes to the state of an execution. <br />
-:dna: **Data lineage**: Track the movement and transformation of data throughout the lifecycle of your data and ML workflows. <br />
-:chart_with_upwards_trend: **Data visualization**: Visualize data, monitor models and view training history through plots. <br />
-:factory: **Dev to prod**: As simple as changing your [domain](https://docs.flyte.org/en/latest/concepts/domains.html) from development or staging to production. <br />
-:money_with_wings: **Spot or preemptible instances**: Schedule your workflows on spot instances by setting `interruptible` to `True` in the task decorator. <br />
-:cloud: **Cloud-native deployment**: Deploy Flyte on AWS, GCP, Azure and other cloud services. <br />
-:calendar: **Scheduling**: [Schedule](https://docs.flyte.org/projects/cookbook/en/latest/auto_examples/productionizing/lp_schedules.html) your data and ML workflows to run at a specific time. <br />
-:mega: **Notifications**: Stay informed about changes to your workflow's state by configuring [notifications](https://docs.flyte.org/projects/cookbook/en/latest/auto_examples/productionizing/lp_notifications.html) through Slack, PagerDuty or email. <br />
-:hourglass: **Timeline view**: Evaluate the duration of each of your Flyte tasks and identify potential bottlenecks. <br />
-:fast_forward: **GPU acceleration**: Enable and control your tasks’ GPU demands by requesting resources in the task decorator. <br />
-:whale: **Dependency isolation via containers**: Maintain separate sets of dependencies for your tasks so no dependency conflicts arise. <br />
-:arrows_counterclockwise: **Parallelism**: Flyte tasks are inherently parallel to optimize resource consumption and improve performance. <br />
-:floppy_disk: **Allocate resources dynamically** at the task level. <br />
-:play_or_pause_button: [Wait](https://docs.flyte.org/projects/cookbook/en/latest/auto_examples/advanced_composition/waiting_for_external_inputs.html) for **external inputs** before proceeding with the execution. <br />
+<img alt="Getting started with Flyte, showing the welcome screen and Flyte dashboard" src="https://raw.githubusercontent.com/flyteorg/static-resources/main/flytesnacks/getting_started/getting_started_console.gif" style="width: 100%; height: auto;" />
```

Comment:
we should probably place this GIF below `pyflyte run --remote hello_world.py hello_world_wf` command.